packages/bpk-component-calendar/README.md

```diff
@@ -16,8 +16,8 @@ import BpkCalendar from 'bpk-component-calendar';
 import BpkInput, { INPUT_TYPES } from 'bpk-component-input';
 import format from 'date-fns/format';
 
-const formatDateFull = date => format(date, 'dddd, Do MMMM YYYY');
```

Comment:
Anywhere you see a change in the format like this, it's because 1.x was using a slightly custom format, and in 2.x they updated it to be consistent with ISO formatting for dates. It doesn't change what users see, as evidenced by the snapshots not changing.